library/nodeping.py

```diff
@@ -327,6 +327,13 @@
   nodeping:
     action: delete
     checkid: 201205050153W2Q4C-0J2HSIRF
+
+# Disable a check
+- name: Disable this check based on its target
+  nodeping:
+    action: disable
+    target: https://example.com
+    enabled: no
```

Comment:
Ah I just looked at the nodeping_maintenance module: It only supports creating maintenance windows.

Reply:
I know https://github.com/NodePing/nodeping-ansible but since we didn't fork this in the first place I did change our copy of it.
Disabling a check can't be done by it's ID. I know it's very confusing.

Reply:
Yes we didn't fork it, because we didn't change it. We just included a copy of it to make it easier to use the role.